library/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,14 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
 
-          package="com.mindera.skeletoid"
-        >
+    package="com.mindera.skeletoid">
 
+    <provider
```

Comment:
Should we add this, shouldn't it be the user to set this up?
When the manifest is merged, it will always put this provider, but the dev may not want to use it